projojo_frontend/src/components/TaskCard.jsx

```diff
@@ -7,7 +7,7 @@ export default function TaskCard({ task }) {
     const totalRegistered = task.totalRegistered || 0;
     const totalNeeded = task.totalNeeded || task.total_needed || 1;
     const title = task.title || task.name;
-    const skills = task.skills || [];
+    const skills = task.skills || task.skills || [];
```

Comment:
```suggestion
    const skills = task.skills  || [];
```
waarom heb je twee keer task.skills ?